### PR TITLE
Access Avatars component

### PR DESCRIPTION
**Pull request by csiyang**
- Created AccessAvatars component
- Added stories

### CODE REVIEW

File: apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+const AccessAvatarsDemo = {
+  ...journeyAdminConfig,
+  component: AccessAvatars,
+  title: 'Journey-Admin/JourneyList/JourneyCard/AccessAvatar'
+}
+
+const Template: Story = ({ ...args }) => (
+  <>
+    <AccessAvatars users={args.users} />
```
Comment (storyworks) [inline apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx:20]:
```suggestion
    <AccessAvatars {...args} />
```

File: apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  </>
+)
+
+export const Single: Story<AccessAvatarsProps> = Template.bind({})
+Single.args = singleAvatar
```
Comment (storyworks) [inline apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx:25]:
```suggestion
Single.args = {
users: [ singleAvatar ]
}
```

File: apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx (hunk 1 above)
Comment (csiyang) [reply apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx]: updated as suggested

File: apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx (hunk 2 above)
Comment (csiyang) [reply apps/journeys-admin/src/components/blocks/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.stories.tsx]: updated as suggested

File: apps/journeys-admin/pages/journeys/[journeySlug].tsx [hunk 3]
```diff
@@ -73,3 +72,24 @@ export const getServerSideProps: GetServerSideProps<SingleJourneyPageProps> =
   }
 
 export default SingleJourneyPage
+
```
Comment (tataihono) [inline apps/journeys-admin/pages/journeys/[journeySlug].tsx:75]: remove.

**Review by tataihono — CHANGES_REQUESTED** [review]
this PR should be returned to draft an be implemented post changes coming from @GeronimoJohn and @gdub01

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+import { GetJourneys_journeys_userJourneys_user } from '../../../../../__generated__/GetJourneys'
+
+export interface AccessAvatarsProps {
+  users: Array<GetJourneys_journeys_userJourneys_user | null>
+}
+
+export function AccessAvatars({ users }: AccessAvatarsProps): ReactElement {
+  const breakpoints = useBreakpoints()
+
+  const maxAvatars = breakpoints.sm ? 5 : 3
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.tsx:13]:
```suggestion
  const maxAvatars = breakpoints.md ? 5 : 3
```

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    }
+
+    const { getByRole, getByText } = render(
+      <AccessAvatars users={props.users} />
+    )
+    expect(getByText('+2')).toBeInTheDocument()
+    fireEvent.focus(getByText('+2'))
+    await waitFor(() => {
+      expect(getByRole('tooltip')).toBeInTheDocument()
+      expect(getByText('Janelle Clegg')).toBeInTheDocument()
```
[the remark this answers is not in the document]
Comment (csiyang) [reply apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.spec.tsx:41]: Changed the lastNames to respective numbers

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.spec.tsx [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+      )
+      expect(getByText('+2')).toBeInTheDocument()
+      fireEvent.focus(getByText('+2'))
+      await waitFor(() => {
+        expect(getByRole('tooltip')).toBeInTheDocument()
+        expect(getByText('Janelle Five')).toBeInTheDocument()
+        expect(getByText('Drake Six')).toBeInTheDocument()
+      })
+    })
+    it('should display 5 avatars max for desktop', () => {
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.spec.tsx:62]:
```suggestion
    it('should display 5 avatars max', () => {
```

**Review by storyworks — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.spec.tsx [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      )
+      expect(getByText('+2')).toBeInTheDocument()
+      fireEvent.focus(getByText('+2'))
+      await waitFor(() => {
+        expect(getByRole('tooltip')).toBeInTheDocument()
+        expect(getByText('Janelle Five')).toBeInTheDocument()
+        expect(getByText('Drake Six')).toBeInTheDocument()
+      })
+    })
+    it('should display 5 avatars ', () => {
```
Comment (storyworks) [inline apps/journeys-admin/src/components/JourneyList/JourneyCard/AccessAvatars/AccessAvatars.spec.tsx:62]:
```suggestion
    it('should display 5 avatars max', () => {
```

File: apps/journeys-admin/pages/journeys/[journeySlug].tsx [hunk 8]
```diff
@@ -78,56 +78,57 @@ function JourneyViewPage({ journey }: JourneyViewPageProps): ReactElement {
...
-export const getServerSideProps: GetServerSideProps<JourneyViewPageProps> =
-  async (context) => {
-    const { data } = await client.query<GetJourney>({
-      query: gql`
-        ${BLOCK_FIELDS}
-        ${INVITE_USER_MODAL_FIELDS}
-        query GetJourney($id: ID!) {
-          journey(id: $id, idType: slug) {
+export const getServerSideProps: GetServerSideProps<
+  JourneyViewPageProps
```
Comment (storyworks) [inline apps/journeys-admin/pages/journeys/[journeySlug].tsx:82]: These haven't changed at all, how come they've been reformatted?

Happens to a couple of files in this..

File: apps/journeys/next-env.d.ts [hunk 9]
```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```
Comment (storyworks) [inline apps/journeys/next-env.d.ts:2]: This is still removed for some reason...

**Review by storyworks — CHANGES_REQUESTED** [review]

File: apps/journeys/next-env.d.ts (hunk 9 above)
Comment (csiyang) [reply apps/journeys/next-env.d.ts:2]: I'll add that line of code back in

File: apps/journeys-admin/pages/journeys/[journeySlug].tsx (hunk 8 above)
Comment (csiyang) [reply apps/journeys-admin/pages/journeys/[journeySlug].tsx:82]: Not sure, I haven't touched them at all. I'll change it back though

Comment (csiyang) [reply apps/journeys-admin/pages/journeys/[journeySlug].tsx:82]: @storyworks Found out these changes are brought in by prettier

**Review by storyworks — APPROVED** [review]

**Pull request merged**